src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,112 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+type Props = {
+  onAdd: (movie: Movie) => void;
+};
+
+export const NewMovie: React.FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
+  const [validImg, setValidImg] = useState(true);
```

Comment:
```suggestion
  const [isValidImg, setIsValidImg] = useState(true);
```
![image](https://github.com/mate-academy/react_movies-list-add-form/assets/96409563/25ec677c-402b-46b7-bd33-fcbd70fc5bea)
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig